challenges/platform/README.md

```diff
@@ -0,0 +1,59 @@
+2. [Troubleshooting challenge](troubleshooting.md)
+
+### Levels
+
+
+| Level  | Oncall  | Operations  | Security  | Communications / Writing  | Networking  | Systems Engineering  | OS Understanding  | Tooling |
+|---|---|---|---|---|---|---|---|---|
+| 1 | Participates in on-call and able to execute run books | Documents and Executes maintenance activities in production | Able to follow security guidelines and best practices in a production environment  | Contributor to internal documentation such as updating run books | Able to troubleshoot introductory networking issues, network policies, routing, etc. | Focus on and contribute to specific area of production system (such as the deployment of our monitoring stack)  | Working understanding of core OS concepts, filesystem, networking, cgroups, namespaces, security mechanisms, etc.  | Work with team members on deployment and maintenance of internal tooling |
+| 2 | Can investigate and solve a majority of production incidents and issues with peer review  | Develops and tests maintenance of production environment (backup/restore as an example)  | Able to analyze and secure new tools or systems connectivity  | Write new documentation, run-books, guides. Able to lead knowledge sharing sessions  |   | Design, write, and test automation for different environments, such as chat bots, upgrade scripts, etc. |   | Able to take ownership and key contributor to key tooling.|
+| 3 | Provide leadership, guidance, and act as escalation point for junior team members |   | Take charge of security incident coordination and create recommendations based on new findings | Contribute to engineering blog, meetups, booth duty, etc to advertise product and engineering culture at Teleport  | Solid understanding of advanced networking, routing, load balancing concepts and scaling production systems  | Design and implement significant automation for challenging problem spaces, such as implementing horizontal/vertical autoscaling and resource allocation | Solid understanding of workload isolation, syscall filtering, linux internals, for high security and production workloads  | Take design lead on key operational areas, such as introducing new tooling, new monitoring, or complex automation |
+| 4 | Helps set and guide the production culture for the team | Finds the areas we are weak and takes ownership on solutions  | Lead security design work for meeting compliance, engineering, or team goals  |   | Able to design solutions that cross team boundaries that make teleport cloud a best in class service  |   |   | Able to work with vendors or opensource communities to ensure external tooling meets our security, compliance, and operational needs |
+
+## Automation / CICD Engineering / Tools / Force Multiplier
+TBD - Use Devops challenge for now
+
+## Cloud Engineering
+New Challenge TBD
+
+## DevSecOps
+TBD - Use Devops challenge for now, security challenge TBD
+
+## Security Engineering
+TBD
```

Comment:
YAGNI -- lets not hit at or promise things before they're ready -- it presents a less professional front.